### ISSUE TITLE
Ye the bot's ded in many servers atm

### ISSUE THREAD

**Issue by amianju** [poketwo/poketwo#366]
Ye the bot's ded in many servers atm

_Originally posted by @LoneWolfisop in https://github.com/poketwo/poketwo/issues/365#issuecomment-1017059122_

**Comment by amianju** [comment]
the bot is online but it isn't reacting to most of the commands, that is interesting

**Comment by VarMonke** [comment]
Yes, we know. We're investigating to find the root of the problem. Please check [#bot-outages](https://discord.com/channels/716390832034414685/717972767252676629/933743288186454016) of the official server for news regarding the bot being down.
